llvm/lib/CodeGen/MachineSSAContext.cpp

```diff
@@ -84,6 +84,26 @@ bool MachineSSAContext::isConstantOrUndefValuePhi(const MachineInstr &Phi) {
   return true;
 }
 
+template <>
+void MachineSSAContext::getPhiInputs(
+    const MachineInstr &Phi, SmallVectorImpl<Register> &Values,
+    SmallVectorImpl<const MachineBasicBlock *> &Blocks) const {
+  if (!Phi.isPHI())
```

Comment:
Surely you can assert this? It would seem weird to define getPhiInputs any other way.